features/step_definitions/survey_steps.rb

```diff
@@ -58,11 +64,18 @@
   visit '/surveys'
 end
 
-#done
-# When( /^I click on the Take survey link in row (\d+)$/) do |link_text, row_number|
-#   within("table tr:nth-child(#{row_number}) td:first-child") do
-#     click_link link_text
-#   end
+When(/^I click on the "Take Survey" link on the first row$/) do
+  within("table tbody tr:nth-child(1)") do
+    click_on "Take Survey"
+  end
+end
+
+Then(/^I should see the "English", "Spanish", and "Chinese" links$/) do
+  expect(page).to have_link("English")
+  expect(page).to have_link("Spanish")
+  expect(page).to have_link("Chinese")
+end
+
 # #done
```

Comment:
There is quite a bit of commented out code in all of these files -> lets delete whatever we do not need, if we may potentially need something in the future add a comment explaining why we are keeping that code!